Cython/Debugger/libcython.py

```diff
@@ -117,6 +117,61 @@ def wrapper(self, string, *args, **kwargs):
 # Classes that represent the debug information
 # Don't rename the parameters of these classes, they come directly from the XML
 
+def simple_repr(self, renamed={}, skip=[], state=True):
```

Comment:
Let's avoid mutable default arguments.
```suggestion
def simple_repr(self, renamed=None, skip=(), state=True):
```